src/sign-verify/clu_x509_verify.c

```diff
@@ -25,6 +25,8 @@
 #include <wolfclu/sign-verify/clu_verify.h>
 #include <wolfclu/x509/clu_cert.h>
 
+#ifndef WOLFCLU_NO_FILESYSTEM
```

Comment:
with WOLFCLU_NO_FILESYSTEM we should have a --disable-filesystem (matching the wolfSSL version)